package.json

```diff
@@ -2,29 +2,23 @@
   "name": "monorepo",
   "private": true,
   "scripts": {
-    "typecheck": "turbo typecheck",
-    "build": "turbo build",
-    "test": "turbo test",
-    "lint": "turbo lint",
-    "format": "prettier --write \"**/*.{ts,tsx,md}\"",
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏I think separated commands might be useful too

Reply:
ok, I'll put them back!